src/gdcdictionary/schemas/sample.yaml

```diff
@@ -747,6 +747,7 @@ properties:
       - FFPE
       - Fresh
       - Frozen
+      - EDTA
```

Comment:
Should an enumDef be added for this?

Reply:
None of the other specific values have one, but it does look a [term](https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C61742) does exist; we would have to update the terms YAML first to not break anything with this PR though.

Reply:
Understood, thank you all for the feedback!